Documentation/ExtensionArchitecture/HowTo/BackendModule/ModuleConfiguration.rst

```diff
@@ -0,0 +1,311 @@
+.. include:: /Includes.rst.txt
+.. index::
+   Backend modules; configuration
+   File; Configuration/Backend/Modules.php
+.. _backend-modules-configuration:
+
+============================
+Backend module configuration
+============================
+
+.. versionchanged:: 12.0
+   Registration of backend modules was changed with Version 12. If you are
+   using another version of TYPO3 please use the version switcher on the top
+   left of this document to go to the respective version.
+
+The configuration of backend modules is placed in the
+dedicated :file:`Configuration/Backend/Modules.php` configuration file.
+
+Those files are then read and processed when building the container. This
+means the state is fixed and cannot be changed at runtime.
+
+Example: register two backend modules
+=====================================
+
+You can find the following example in
+`EXT:examples <https://github.com/TYPO3-Documentation/t3docs-examples>`__.
+
+Two backend modules are being registered. The first module is based on
+Extbase while the second uses a plain controller.
+
+.. code-block:: php
+   :caption: EXT:my_extension/Configuration/Backend/Modules.php
+
+   use T3docs\Examples\Controller\ModuleController;
+   use T3docs\Examples\Controller\AdminModuleController;
+
+   /**
+    * Definitions for modules provided by EXT:examples
+    */
+   return [
+       'web_examples' => [
+           'parent' => 'web',
+           'position' => ['after' => 'web_info'],
+           'access' => 'user',
+           'workspaces' => 'live',
+           'path' => '/module/page/example',
+           'labels' => 'LLL:EXT:examples/Resources/Private/Language/Module/locallang_mod.xlf',
+           'extensionName' => 'Examples',
+           'controllerActions' => [
+               ModuleController::class => [
+                   'flash','tree','clipboard','links','fileReference','fileReferenceCreate',
+               ],
+           ],
+       ],
+       'admin_examples' => [
+           'parent' => 'system',
+           'position' => ['top'],
+           'access' => 'user',
+           'workspaces' => 'live',
+           'path' => '/module/system/example',
+           'labels' => 'LLL:EXT:examples/Resources/Private/Language/AdminModule/locallang_mod.xlf',
+           'extensionName' => 'Examples',
+           'controllerActions' => [
+               AdminModuleController::class => [
+                   'index',
+               ],
+           ],
+       ],
+   ];
+
+
+Module configuration options
+============================
+
+.. confval:: parent
+
+   :Scope: Backend module configuration
+   :type: string
+
+   If the module should be a submodule, the parent identifier, for example `web`
+   has to be set here. You can find a
+   :ref:`list of available toplevel modules. <backend-modules-toplevel-module>`
+
+   Extensions can add additional parent modules by defining
```

Comment:
This is not really correct. I can add also new parent modules with submodules, not only standalone modules (like dashboard).